Python/frame.c

```diff
@@ -118,6 +118,20 @@ _PyFrame_Clear(_PyInterpreterFrame *frame)
     Py_DECREF(frame->f_code);
 }
 
+/* Limited API functions */
+
+PyCodeObject *_PyInterpreterFrame_GetCode(struct _PyInterpreterFrame *frame)
+{
+    PyCodeObject *code = frame->f_code;
+    Py_INCREF(code);
+    return code;
```

Comment:
```suggestion
    return (PyCodeObject *)Py_NewRef(frame->f_code);
```

Reply:
I like to put the code on a single line, but it's up to you.

For me, it became easier to use Py_NewRef() if I think that the result is a new reference. For me, Py_INCREF() is a really strange API: it takes a borrow reference and converts it to a strong reference in-place. Doc: https://docs.python.org/dev/c-api/refcounting.html#c.Py_INCREF

But I know that most core devs are use to the classic Py_INCREF() way ;-)